components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/auth/SCIMAuthConfigReader.java

```diff
@@ -91,4 +91,24 @@
         }
         return Collections.emptyList();
     }
+
+    /**
+     * Get the configuration value from identity.xml file.
+     *
+     * @return true or false
+     */
+    public boolean isGetAllDetailsEnabled() {
```

Comment:
Is it necessary to use this method to read identity.xml configuration?. Can't you use IdentityUtil.getProperty() to read the value from identity.xml?